_pages/fr/codes-source-ouverts.html

```diff
@@ -5,13 +5,19 @@
 permalink: /fr/codes-source-ouverts.html
 ---
 
-<h1 property="name" id="wb-cont">Code source ouvert</h1>
-<p>Projets libres et dépôts de code des administrations publiques Canadiennes.</p>
-<p>Consultez la liste des <a href="./code-source-ouvert-groupes.html">Groupes de code source ouvert</a> pour trouver plus de projets libres et dépôts de code source ouvert d'administrations publiques Canadiennes.</p>
-<p>Consultez la <a href="{{ site.data.i18n.general.nav.doc.link[page.lang] }}">{{ site.data.i18n.general.nav.doc[page.lang] }}</a> et le <a href="./code-source-ouvert-schema.html">Schéma de métadonnées du code source ouvert</a> pour la liste des champs et comment ajouter vos données.</p>
+<h1 property="name" id="wb-cont">Code Source Ouvert</h1>
+<p>
+  Le code source ouvert fait référence au logiciel dont le développement suit un processus ouvert. La licence du projet permet d’accéder librement au code source, de créer des travaux dérivés et de le redistribuer.
+  Ce processus de développement ouvert est généralement le résultat de la collaboration entre plusieurs parties prenantes pour l’élaboration de solutions technologiques à des problèmes communs.
+</p>
+
+<h2 property="name">Projets</h2>
+<p>
+  Consultez la liste des différents projets ouverts élaborés par des administrations publiques canadiennes du niveau fédéral, provincial, municipal et autochtone.
+  Vous pouvez modifier les informations d’un projet existant dans la liste ou ajouter un nouveau projet ouvert en soumettant le <a href="./code-source-ouvert-formulaire.html">Formulaire de Code Source Ouvert</a>.<br>
+  <strong>Besoin d’aide?</strong> Consulter la documentation <a href="#">comment utiliser la plateforme Échange de Ressources Ouvert</a> pour plus d’informations sur comment bénéficier de ce service.
```

Comment:
"Consulter" should be "Consultez"

Also "Consultez la documentation comment utiliser..." doesn't make sense as a sentence, I feel like words are missing.

Reply:
Outdated